src/lib/y2storage/filesystems/legacy_nfs.rb

```diff
@@ -95,9 +100,8 @@ def to_hash
           "device"  => share_string(server, path),
           "mount"   => mountpoint,
           "fstopt"  => fstopt,
-          # TODO: libstorage-ng does not distinguish different NFS versions, so
-          # for the time being, always use :nfs here
-          "used_fs" => :nfs
+          "used_fs" => fs_type.to_sym,
```

Comment:
Please add a comment what the difference between the two is (which is which).

Reply:
Great question :-)